src/org/zaproxy/zap/extension/api/PhpAPIGenerator.java

```diff
@@ -119,7 +119,7 @@ private void generatePhpElement(ApiElement element, String component,
 		String paramOpt = "";
 		if (element.getOptionalParamNames() != null) {
 			for (String param : element.getOptionalParamNames()) {
-			    if (paramMan != "" || paramOpt != "") {
+			    if (paramMan.length() > 0 || paramOpt.length() > 0) {
```

Comment:
Not sure worth using isBlank, it's never null and it's either empty or has non whitespace chars.